float32[6] Op